### PR TITLE
Fix contact import customize

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/contacts/tests.py [hunk 1]
```diff
@@ -1594,6 +1600,8 @@ def test_contact_import(self):
         self.assertEquals(contact1.get_field_raw('location'), 'Rwanda')  # renamed from 'Country'
         self.assertEquals(contact1.get_field_display('location'), 'Rwanda')  # renamed from 'Country'
 
+        self.assertEquals(contact1.get_field_raw('ride_or_drive'), 'Moto') # the existing field was looked up by label
+
```
Comment (nicpottier) [inline temba/contacts/tests.py]: Maybe also have a test case with some extra spaces? IE, "Moto "

Comment (norkans7) [inline temba/contacts/tests.py]: I realize both "Vehicle" and "Vehicle " are both valid labels. should that be prevented?

File: temba/contacts/models.py [hunk 2]
```diff
@@ -59,7 +59,7 @@ def is_valid_key(cls, key):
 
     @classmethod
     def is_valid_label(cls, label):
-        return regex.match(r'^[A-Za-z0-9\- ]+$', label, regex.V0)
+        return regex.match(r'^[A-Za-z0-9\-][A-Za-z0-9\- ]+[A-Za-z0-9\-]$', label, regex.V0)
```
Comment (nicpottier) [inline temba/contacts/models.py]: I assume this is trying to enforce that labels don't start or end with spaces, but it is also forcing labels to be at least 3 characters.

Wouldn't having a strip() here accomplish the same thing?

File: temba/contacts/views.py [hunk 3]
```diff
@@ -1010,7 +1019,8 @@ def clean(self):
 
                 if label:
                     if not ContactField.is_valid_label(label):
-                        raise forms.ValidationError(_("Field names can only contain letters, numbers, spaces and hypens"))
+                        raise forms.ValidationError(_("Field names can only contain letters, numbers, "
+                                                      "hypens and not trailling spaces"))
```
Comment (nicpottier) [inline temba/contacts/views.py]: Again, I would think we just strip the trailing spaces, so no need to mention them.

File: temba/contacts/views.py [hunk 4]
```diff
@@ -324,11 +324,12 @@ def clean(self):
...
                 for key, value in self.data.items():
                     if re_col_name_field.match(key):
-                        field_label = value
+                        field_label = value.strip()
                         field_key = slugify_with(value)
 
                         if not ContactField.is_valid_label(field_label):
-                            raise ValidationError(_("Field names can only contain letters, numbers, spaces and hypens"))
+                            raise ValidationError(_("Field names can only contain letters, numbers, "
+                                                    "hypens and not trailling spaces"))
```
Comment (nicpottier) [inline temba/contacts/views.py]: strip.

File: temba/contacts/tests.py [hunk 5]
```diff
@@ -1557,13 +1557,18 @@ def test_contact_import(self):
         self.assertEquals(Contact.objects.all().count(), 0)
         self.assertEquals(ContactGroup.user_groups.all().count(), 0)
 
+        # existing field
+        ContactField.get_or_create(self.org, 'ride_or_drive', 'Vehicle')
```
Comment (nicpottier) [inline temba/contacts/tests.py]: get_or_create should be modified to just strip()

**Comment by nicpottier** [comment]
yep

**Pull request merged**